src/pages/mentees.tsx

```diff
@@ -111,8 +111,8 @@ function MenteeTable({ users, refetch }: {
         const yearB = menteeToYear.get(b.id) || "";
 
         if (yearA === yearB) {
-          const nameA = formatUserName(a.name, 'formal');
-          const nameB = formatUserName(b.name, 'formal');
+          const nameA = toPinyin(formatUserName(a.name, 'formal'));
```

Comment:
why need to change this?

Reply:
I see. In this case, add a function compareChinese in strings.ts similar to compareDate. Also add this link to the comment of this function to justify its existence.

Reuse this compareChinese in the `Applicants` function